src/ts-framework/Core/Application.ts

```diff
@@ -16,7 +16,7 @@ export class Application
      * Default port used by the application in development mode
      * @type {number}
      */
-    public static DEFAULT_PORT = 3000;
+    public static DEFAULT_PORT = process.env.PORT || 3000;
```

Comment:
The port can be changed via `Application#start(port: number)` or by specifying it in the config, this line should be moved there instead, `DEFAULT_PORT` is a constant which shouldn't be changed.